tickit/adapters/interpreters/command/regex_command.py

```diff
@@ -36,24 +36,50 @@ def __call__(self, func: Callable) -> Callable:
         setattr(func, "__command__", self)
         return func
 
-    def parse(self, data: bytes) -> Optional[Sequence[AnyStr]]:
+    def parse(self, data: AnyStr) -> Optional[Tuple[Sequence[AnyStr], int, int, int]]:
```

Comment:
This can be drastically simplified by removing all encoding / decoding if we merge #91 first